projects/angular-calendar/test/calendar-month-view.component.spec.ts

```diff
@@ -29,6 +29,10 @@ import { Subject } from 'rxjs';
 import { triggerDomEvent } from './util';
 import { take } from 'rxjs/operators';
 import { adapterFactory } from '../src/date-adapters/date-fns';
+import localeDe from '@angular/common/locales/de';
```

Comment:
How come this is needed? I thought by default angular ships with the english translation so it shouldn't trigger any errors?

Reply:
One of the tests in this file failed in Travis CI until the german locale was loaded here. What is strange is that similar tests for other components did not fail when the german locale wasn't loaded.

Reply:
Oh how weird! OK, let's just leave it in then